core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -229,4 +231,15 @@ public Duration getTenantIdleTimeout() {
     public void setTenantIdleTimeout(final Duration tenantIdleTimeout) {
         this.tenantIdleTimeout = Objects.requireNonNull(tenantIdleTimeout);
     }
+
+    /**
+     * Gets the configured mappers for this adapter
+     * <p>
+     * The default value of this property is an empty hashMap. Ths results in not having custom mapping at all.
+     *
+     * @return HashMap of mapper endpoints
+     */
+    public HashMap<String, MapperEndpoint> getMapperEndpoints() {
```

Comment:
can you also add a setter so that this can be used in unit tests without requiring reflection?

Reply:
please also make the getter and setter final